pkg/cosign/remote/remote.go

```diff
@@ -70,7 +70,8 @@ func DockerMediaTypes() bool {
 func SignatureImage(ref name.Reference, opts ...remote.Option) (v1.Image, error) {
 	base, err := remote.Image(ref, opts...)
 	if err != nil {
-		if te, ok := err.(*transport.Error); ok {
+		te := &transport.Error{}
```

Comment:
Can these be inlined?

Reply:
I tried that, but I think because of the `{}` it cannot 😞

Reply:
Now they are vars, so really no.